spec/DemographicData.csvschema.json

```diff
@@ -0,0 +1,9 @@
+{
+    "columns": {
+        "country": "String",
+        "age": "String",
+        "year": "Int",
+        "gender": "String",
+        "value": "Decimal"
+    }
+}
```

Comment:
Is a string age range really the best way of communicating the age range? Wouldn't two columns `ageFirst` and `ageLast`, as in the coverage data, be better? Otherwise anyone consuming this programmatically has to parse the "this - that" format. Also, it's a little unclear whether the upper age bound is inclusive or exclusive.

Sorry if this is something that has been discussed to death with Wes and Tini already.